opencti-platform/opencti-front/src/private/components/data/import/ImportFilesContent.tsx

```diff
@@ -179,7 +179,7 @@ const ImportFilesContent = () => {
         mode: 'or',
       },
       {
-        key: 'entity_id',
+        key: 'metaData.entity_id',
```

Comment:
I don't understand how we could delete all uploaded files (not only global) since we have also this filter in toolbarFilters : 
`{
        key: 'file_id',
        values: ['import/global'],
        operator: 'starts_with',
      }`
Does this filter work ?